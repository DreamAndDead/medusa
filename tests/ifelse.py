a = 45
b = 0

if a > 5 and b < 34:
    print("a > 5")
    if a >= 45:
        print("a >= 45")
    else:
        print("a < 45")
elif a < 5:
    print("a < 5")
else:
    print("a == 5")

if a == 45:
    print("a == 45")

x = 100
if 50 < x < 150:
    print("50 < x < 150")
else:
    print("Something wrong.")