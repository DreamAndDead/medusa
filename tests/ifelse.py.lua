dofile('./pythonlua/luainit.lua')

local a = 45
local b = 0
if ((a > 5) and (b < 34)) then
    print("a > 5")
    if (a >= 45) then
        print("a >= 45")
    else
        print("a < 45")
    end
elseif (a < 5) then
    print("a < 5")
else
    print("a == 5")
end
if (a == 45) then
    print("a == 45")
end
local x = 100
if (50 < x and x < 150) then
    print("50 < x < 150")
else
    print("Something wrong.")
end