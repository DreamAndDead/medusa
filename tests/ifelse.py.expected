a > 5
a >= 45
a == 45
50 < x < 150
